### DISCUSSION TITLE
the program "depmod" was not found in path...

### DISCUSSION THREAD

**Discussion by slasxxx** [Freetz-NG/freetz-ng#253] [Allgemein]
Hallo, ich habe ein Problem beim bauen der Firmware...

Genutzt wird das aktuellste Freetz-NG Image: Freetz-Linux-1.5.8-Ubuntu 20.04.01 LTS (Focal Fossa) 64-Bit
Inkl. aller Updates etc.

Leider kommt beim erstellen der Fehler: the program "depmod" was not found in path... Ich habe alle prequesties installiert...

Kann mit bitte jemand helfen?

**Comment by Conan179** [comment]
Was sagt "sudo apt install depmod" genau bei dir?

**Reply by slasxxx** [reply]
Paket depmod kann nicht gefunden werden

**Reply by Conan179** [reply]
Screenshot bitte

**Reply by slasxxx** [reply]
![Screenshot 2021-04-05 105500](https://user-images.githubusercontent.com/81957255/113556953-8aa0f180-95fd-11eb-986d-9cf0237a827a.png)

**Reply by Conan179** [reply]
probier bitte mal "sudo apt install kmod"

**Reply by slasxxx** [reply]
![2](https://user-images.githubusercontent.com/81957255/113557207-f08d7900-95fd-11eb-9166-0539244b9491.png)

**Reply by Conan179** [reply]
hmm inntresant, depmod kan man scheinbar gar nicht nach innsallieren, es ist einfach da.
Ultimaratio wäre, das du die bau vm neu runterlädst.

**Reply by slasxxx** [reply]
Das habe ich schon 3x versucht... auch auf unterschiedlichen Hosts :/ Ich verstehe es einfach nicht mehr. Danke für deine Hilfe

**Reply by Conan179** [reply]
we du 3 mal die selbe version runtergleaden ahst (host egal) würde ich mal eine ältere version nehmen.

**Reply by slasxxx** [reply]
alles klar, ich versuche es nochmal... Melde mich sobald ich durch bin :)

**Comment by slasxxx** [comment]
![3](https://user-images.githubusercontent.com/81957255/113557384-38140500-95fe-11eb-981e-960b83dfe271.png)

Screenshot von dem Problem beim make

**Comment by Gismotro** [comment]
Bitte lesen: https://github.com/Freetz-NG/freetz-ng/discussions/235

**Comment by slasxxx** [comment]
danke, ist erledigt! Habe es mit einer neuen Ubuntu versucht ;)

**Reply by Gismotro** [reply]
??? Neue VM gebaut ?
